src/app/sitemap.ts

```diff
@@ -0,0 +1,19 @@
+import { CATEGORIES, SITE_URL } from '@/constants';
+import { MetadataRoute } from 'next';
+
+const createPages = (urls: string[]) => {
```

Comment:
In my previous sample code I called it `paths`, because these are partial urls and it can be confusing.

Why do you prefer calling it `pages` ?

Reply:
Sorry I was focusing on the structure, will make the names similar too.